website/docs/style_guide/about_style_guide.md

```diff
@@ -0,0 +1,65 @@
+# Robot Framework Style Guide
+
+## Purpose
+
+This style guide is a community driven set of sensible rules to write your Robot Framework code.
+
+As a starting point the existing standards from [Robot Framework](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#getting-started) user guide, [Robocop](https://github.com/MarketSquare/robotframework-robocop) and [Robotidy](https://robotidy.readthedocs.io/en/stable/).
+
+## Robocon 2022 Presentation
+
+[Robot Framework is not ...](https://youtu.be/Mpt_4MItha0)
+
+## Contributors
+
+Guido Demmenie, Mannana Koberidze, Kelby Stine
+
+### Special Thanks To
+
+Václav Fuksa, Many Kasiriha, Bartlomiej Hirz, Mateusz Nojek, René Rohner, Miikka Solmela
+
+### We are always looking for contributors
+
+- We meet up once every two weeks to talk about style guide topics
+- Reach out to any one of us on the Slack channels
+
+---
+
+## Basic Rules and Guidelines
+
+Here are some good basic guidelines for formatting your Robot Framework codes. As these are guidelines, use your best judgment in cases not covered by these recommendations.
+
+- Separate pieces of the data, such as keywords and their arguments, with four (4) spaces.
+
+- Robot and Resource file section order:
+
+```robot
+*** Comments ***
```

Comment:
Do we really want to have Comments Section at the top? I was always thinking about it as the last one at the bottom, but I don't have a strong opinion here.

Reply:
Believe it or not Comments seem to land at the top with robotidy default settings.

Reply:
Right! Then it's fine :)

Reply:
I didn't see this PR, but for robotidy transforming the order of the sections - it's because section without header is comment section, ie.:

```
# language: pl

*** Settings ***
```

The part before *** Settings *** is actually *** Comments ***. That's why I have decided to put it first (but it was just my decision)